src/content/docs/apm/agents/java-agent/instrumentation/java-agent-instrument-kafka-message-queues.mdx

```diff
@@ -25,6 +26,10 @@ Kafka instrumentation is available in Java agent versions 4.12.0 or higher. For
   For our Kafka integration, see [Kafka monitoring integration](/docs/integrations/host-integrations/host-integrations-list/kafka-monitoring-integration).
 </Callout>
 
+<Callout variant="important">
+  Kafka Streams runs on top of Kafka Clients so all of the instrumentation that applies to Kafka Clients also applies to Streams.
```

Comment:
This isn't clear to me. What instrumentation is this referring to? Do we have something on instrumentation for Kafka Clients somewhere in docs? I didn't see it.

Reply:
Kafka Client is mentioned before in line 23 of the file.